redaxo/src/core/lib/sql/sql.php

```diff
@@ -1618,6 +1618,34 @@ public static function getServerVersion($DBID = 1)
         return self::$pdo[$DBID]->getAttribute(PDO::ATTR_SERVER_VERSION);
     }
 
+    /**
+     * Returns the database type (MySQL or MariaDB).
+     *
+     * @return string `rex_sql::MYSQL` or `rex_sql::MARIADB`
```

Comment:
in blaue geraten.. könnte gehen

```suggestion
     * @return string `rex_sql::MYSQL` or `rex_sql::MARIADB`
     * @psalm-return self::MYSQL|self::MARIADB
```